msal/src/androidTest/res/raw/test_pcaconfig_with_slice.json

```diff
@@ -0,0 +1,21 @@
+{
+  "client_id": "[clientid]",
+  "redirect_uri": "[redirectUri]",
+  "authorities": [
+    {
+      "type": "AAD",
+      "audience": {
+        "type": "AzureADMyOrg",
+        "tenant_id": "[tenantid]"
+      },
+      "slice": {
```

Comment:
@shoatman This is my proposed spec change. `slice` is now an object (map) instead of array of objects

Reply:
Looks good.